.mbedignore

```diff
@@ -1,8 +1,10 @@
-cryptoauthlib/app/
+cryptoauthlib/app/secure_boot/
+cryptoauthlib/app/ip_protecton/
```

Comment:
Please remove changing `cryptoauthlib/app` exclusions in commit "Ignore test untity files". The "Ignore test untity files" commit should only add the ignoring of Unity files.